packages/mds-geography-author-api/tests/api.spec.ts

```diff
@@ -25,15 +25,9 @@
 
 import { ApiServer } from '@mds-core/mds-api-server'
 import db from '@mds-core/mds-db'
-import { GeographyServiceManager } from '@mds-core/mds-geography-service'
-import {
-  DISTRICT_SEVEN,
-  GEOGRAPHY2_UUID,
-  GEOGRAPHY_UUID,
-  LA_CITY_BOUNDARY,
-  POLICY_UUID,
-  SCOPED_AUTH
-} from '@mds-core/mds-test-data'
+import { GeographyServiceClient, GeographyServiceManager } from '@mds-core/mds-geography-service'
+import { GEOGRAPHY2_UUID, GEOGRAPHY_UUID, LA_CITY_BOUNDARY, POLICY_UUID, SCOPED_AUTH } from '@mds-core/mds-test-data'
+import restrictedAreas from '@mds-core/mds-test-data/test-areas/restricted-areas'
```

Comment:
I would try to import this from `@mds-core/mds-test-data` directly if possible (may need to change the structure of test data a lil bit) on the off chance that this test file somehow makes it into the bundle and breaks anything importing this external to mds-core. Probably not _necessary_, but I just had to fix a similar issue a couple weeks ago and it was a PITA.